central/scannerdefinitions/handler/handler.go

```diff
@@ -542,12 +593,13 @@ func (h *httpHandler) handleZipContentsFromVulnDump(ctx context.Context) error {
 	// scanner-defs.zip contains data required by Scanner V2.
 	// scanner-v4-defs-*.zip contains data required by Scanner v4.
 	// In the future, we may decide to support other files (like we have in the past), which is why we
-	// expect this ZIP of a single ZIP.
+	// loop through this ZIP of ZIPs.
```

Comment:
Not really part of your scope... actually, maybe it is, anyway: Offline bundles were supposed to have multiple scanner-v4-defs-* (in particular, one for 4.4. and another for 4.5); [The validation before this call](https://github.com/stackrox/stackrox/blob/master/central/scannerdefinitions/handler/handler.go#L506) is doing the wrong thing and blocking the offline upload if there defs with incompatible changes -- so I think we should verify if there is at least ONE version that is supported, and here we filter out the ones that are not.

Reply:
I was wondering if that was something that would be done as we work on improving our vuln versioning (since it was acknowledged when reviewing the ADR that we know we made a mistake). Were there plans to fix this when working on that or was that unplanned? Perhaps I should just make a ticket to address this and have someone (either myself or someone else) address this later during this development cycle

Reply:
Sounds good. I looked at the current planning for the versioned bundles, and I didn't find a specific task on handling multiple offline bundles in the same offline ZIP; That's probably because the planning was done before the decision of supporting that use-case during the ADR review. So I think it makes sense to create the task and add it to the backward-compatible vulnerabilities (versioned bundle) epic.